src/components/frame/LandingPage/LandingPage.tsx

```diff
@@ -100,16 +105,21 @@ const LandingPage = () => {
   const navigate = useNavigate();
   const { user, connectWallet, wallet, walletConnecting, userLoading } =
     useAppContext();
-  const { setBreadcrumbs } = usePageHeadingContext();
+  const { setRootBreadcrumbItem, setShouldShowBreadcrumbs, resetBreadcrumbs } =
+    useBreadcrumbsContext();
 
   useEffect(() => {
-    setBreadcrumbs([
-      {
-        key: 'landing-page',
-        label: 'Colony App',
-      },
-    ]);
-  }, [setBreadcrumbs]);
+    setRootBreadcrumbItem({
+      link: '',
+      label: formatText(MSG.homeBreadcrumbLabel),
+    });
+    setShouldShowBreadcrumbs(true);
+
+    return () => {
+      resetBreadcrumbs();
+    };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
I think we only need to `setShouldShowBreadcrumbs(false)` here based on the specs